edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -562,73 +562,73 @@ Above all, the competitor limit must not be exceeded for any reason. Competitor
+
+- How to read a nametag, or otherwise determine the competitor's competing, judging, scrambling, and/or running groups.
+- What to do when the competitor's group is called.
+- How to compete.
+  - Telling the judge they are ready.
+  - Inspection time, including penalties.
+  - How to start the timer properly.
+    - Hand position, particularly using fingers to start the timer.
+    - Waiting for the green light on the timer.
+  - How to stop the timer properly.
+    - Flat hands; no "karate chops".
+  - How to sign for attempts.
     - Check that the time matches the timer and is written legibly. Times which are not written legibly will be interpreted in the worst way possible. Competitors can ask their judge to re-write times if they are not easily readable.
-  - How to call over a Delegate
-- How to judge
-  - Asking the competitor if they are ready and waiting for the competitor’s ready signal
-  - Judging inspection, including lifting the cube cover, operating the stopwatch, and making callouts
-    - Emphasize that only "8 seconds" and "12 seconds" are called out, not 15 or 17 seconds
-    - Stopping the stopwatch when the competitor starts their solve, so that the actual time of the inspection is measured
+  - How to call over a Delegate.
+- How to judge.
+  - Asking the competitor if they are ready and waiting for the competitor's ready signal.
+  - Judging inspection, including lifting the cube cover, operating the stopwatch, and making callouts.
+    - Emphasize that only "8 seconds" and "12 seconds" are called out, not 15 or 17 seconds.
+    - Stopping the stopwatch when the competitor starts their solve, so that the actual time of the inspection is measured.
   - What to look for when the competitor starts their solve, including:
     - Timer start hand position
     - Whether the competitor is touching the puzzle
-  - How to judge during the solve
-    - Pay attention. Don’t cube, use your phone, or engage in conversation
-    - Don’t distract the competitor during the solve. Sit still and be quiet
-    - Watch the competitor for any regulation violations
-    - Pay close attention to how the competitor stops their timer, and make sure they are using their hands with palms flat and facing down
-  - Finishing the solve
-    - The first step is to evaluate whether the cube is solved
-    - Do not touch the puzzle, use only your eyes
+  - How to judge during the solve.
+    - Pay attention. Don't cube, use your phone, or engage in conversation.
+    - Don't distract the competitor during the solve. Sit still and be quiet.
+    - Watch the competitor for any regulation violations.
+    - Pay close attention to how the competitor stops their timer, and make sure they are using their hands with palms flat and facing down.
+  - Finishing the solve.
```

Comment:
This is not a complete sentence.  Do not use a period.

Reply:
Do you think we should mix up the use of periods within the same list?

Reply:
I guess that's a good point.  Personally, I'd argue that if one item needs to be a complete sentence, we should have them all be a complete sentence within the same list, but it doesn't make much sense.  Maybe put the "and make sure they are...." in a sub bullet.